### PR TITLE
Rspec matchers

### PR DESCRIPTION
**Pull request by ancorgs**

### CODE REVIEW

**Comment by jreidinger** [comment]
in general it looks good, just how will look error message if failed?

**Comment by ancorgs** [comment]
@jreidinger Fixed, now error messages look like this:
```
Yast::RSpec::Matchers#path_matching does not match
Failure/Error: Yast::SCR.Read(path(".etc"))
  Yast::SCR received :Read with unexpected arguments
    expected: (path_matching(/target.*ir$/))
         got: (#<Yast::Path:0x00000001c07cd0 @components=["etc"]>)
```

**Comment by jreidinger** [comment]
yep, looks better

**Comment by jreidinger** [comment]
just tests do not pass in travis

File: src/ruby/yast/rspec/matchers.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      end
+
+      # @private
+      class PathMatchingMatcher
+        def initialize(expected)
+          @expected = Regexp.new(expected)
+        end
+
+        # RSpec 3 uses === while RSpec 2 uses ==
+        # Thus, implementing just == should work with both
```
Comment (jreidinger) [inline src/ruby/yast/rspec/matchers.rb]: well, maybe add FIXME as it actually is not `==` and `===` is really proper method regarding behavior of class. but just nitpick

**Comment by jreidinger** [comment]
LGTM

**Pull request merged**